hugo/layouts/lookout/single.html

```diff
@@ -34,35 +34,55 @@ <h3 class="title-line line-orange">Introducing source{d} Lookout</h3>
             </div>
         </div>
         <div class="row text-dark mb-5">
-                <div class="col-md-6 col-lg-3 text-center">
+                <div class="col-md-6">
                     <div class="card bg-transparent">
-                        <div class="card-body">
-                            <img class="mb-3" src="img/icons/style-and-patterns.svg" height="140" width="140" alt="">
-                            <p>Keep your code base style/patterns consistent</p>
+                        <div class="card-body row">
+                            <div class="col-sm-3 col-md-12 col-lg-4">
+                                <img class="mb-3 img-fluid mw-150" src="img/icons/style-and-patterns.svg" alt="">
+                            </div>
+                            <div class="col-sm-9 col-md-12 col-lg-8">
+                                <h4 class="title-line line-orange">Keep your code base style/patterns consistent</h4>
+                                <p></p>
+                            </div>
                         </div>
                     </div>
                 </div>
-                <div class="col-md-6 col-lg-3 text-center">
+                <div class="col-md-6">
                     <div class="card bg-transparent">
-                        <div class="card-body">
-                            <img class="mb-3" src="img/icons/language-agnostic-code-reviews.svg" height="140" width="140" alt="">
-                            <p>Language Agnostic assisted code reviews</p>
+                        <div class="card-body row">
+                            <div class="col-sm-3 col-md-12 col-lg-4">
+                                <img class="mb-3 img-fluid mw-150" src="img/icons/language-agnostic-code-reviews.svg" alt="">
+                            </div>
+                            <div class="col-sm-9 col-md-12 col-lg-8">
+                                <h4 class="title-line line-orange">Language Agnostic assisted code reviews</h4>
+                                <p></p>
```

Comment:
leftover?
```suggestion
```

Reply:
In all these cases I left it because it is still missing content, but for sure it can be added once the texts are ready.